dolphin/api/v1/access_info.py

```diff
@@ -37,8 +42,38 @@ def show(self, req, id):
 
         return self._view_builder.show(access_info)
 
+    @validation.schema(schema_access_info.update)
+    def update(self, req, id, body):
+        """Update storage access information."""
+        ctxt = req.environ.get('dolphin.context')
+        # Get existing access_info and storage from DB
+        try:
+            access_info_present = db.access_info_get(ctxt, id)
+            storage_present = db.storage_get(ctxt, id)
+            access_info_updated_dict = access_info_present.to_dict()
```

Comment:
do we need to convert this to dictionary again?. I am talking about to_dict()

Reply:
DB does not return dictionary..

Reply:
agreed. I mean, without converting to dict also, .update(body) should work.

Reply:
+1 Pravin comments, db model supports operation of dict.

Reply:
done